pages/agent/v3/tokens.md

```diff
@@ -1,100 +1,126 @@
+Agent tokens can be revoked using the [_Agent Tokens_ page of a cluster](#revoke-a-token-using-the-buildkite-interface), or via the [REST API's delete agent token](#revoke-a-token-using-the-rest-api) feature.
 
-Tokens can be revoked using the [GraphQL API](/docs/apis/graphql-api) with the `agentTokenRevoke ` mutation.
+Once a token is revoked, no new agents will be able to start with that token. Revoking a token does not affect any connected agents.
 
-You need to pass your agent token as the ID in the mutation. You can get the token from your Buildkite dashboard, in _Agents_ > _Reveal Agent Token_, or you can retrieve a list of agent token IDs using this query:
+### Using the Buildkite interface
 
-```graphql
-query GetAgentTokenID {
-  organization(slug: "organization-slug") {
-    agentTokens(first:50) {
-      edges {
-        node {
-          id
-          uuid
-          description
-        }
-      }
-    }
-  }
-}
-```
+To revoke a cluster's agent token using the Buildkite interface:
+
+1. Select _Agents_ to access the _Agent Clusters_ page.
+1. Select the cluster containing the agent token to revoke.
+1. Select _Agent Tokens_ and on this page, expand the agent token to revoke.
+1. Select _Revoke_ > _Revoke Token_ in the confirmation message.
+
+### Using the REST API
 
-Then, using the token ID, revoke the agent token:
-
-```graphql
-mutation {
-  agentTokenRevoke(input: {
-    id: "token-id",
-    reason: "A reason"
-  }) {
-    agentToken {
-      description
-      revokedAt
-      revokedReason
-    }
-  }
-}
+To [revoke (that is, delete) an agent token](/docs/apis/rest-api/clusters#agent-tokens-delete-a-token) using the [REST API](/docs/apis/rest-api), run the following example `curl` command:
+
+```curl
+curl -H "Authorization: Bearer $TOKEN" \
+  -X DELETE "https://api.buildkite.com/v2/organizations/{org.slug}/clusters/{cluster.id}/tokens/{id}"
 ```
 
-Once a token is revoked, no new agents will be able to start with that token. Revoking a token does not affect any connected agents.
+where:
+
+- The `$TOKEN`, `{org.slug}` and `{cluster.id}` values are obtained the same way as those when [creating an agent token using the REST API](#create-a-token-using-the-rest-api).
+
+- The `{id}` value can be obtained:
+
+    * From the Buildkite URL path when editing the agent token. To do this:
+
+        - Select _Agents_ > the specific cluster > _Agent Tokens_ > expand the agent token > _Edit_.
+        - Copy the ID value between `/tokens/` and `/edit` in the URL.
+
+    * By running the [List tokens](/docs/apis/rest-api/clusters#agent-tokens-list-tokens) REST API query and obtain this value from the `id` in the response associated with the description of your token (specified by the `description` value in the response). For example:
+
+        ```curl
+        curl -H "Authorization: Bearer $TOKEN" "https://api.buildkite.com/v2/organizations/{org.slug}/clusters/{cluster.id}/tokens"
+        ```
 
 ## Scope of access
```

Comment:
When I got to this point in the page, my first thought was that this section and everything below it would be better grouped with 'using and storing' above, and that the very specific instructions for creating and revoking could go below. Then, I reconsidered, because when you're setting up tokens/agents for the first time, the order that you've provided here is probably more practical.

So, at this point, I don't think you should re-order it, but I wanted to share my kinda -- speed-bump thought that happened when I got here, just in case it contributes to your thinking.

Reply:
Yeah, that's a great point. I was thinking that too—placing the 'using and storing' section between the 'create' and 'revoke' sections. However, based on the content of this section, and the fact that one does get an initial agent token upon creating a new org, I think for the time being it makes sense keeping this section near the top.